launcher/image/sync_experiments.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+readonly OEM_PATH='/usr/share/oem'
+readonly CS_PATH="${OEM_PATH}/confidential_space"
+readonly EXPERIMENTS_SCRIPT="experiments_sync"
+readonly SYNC_RESULT_FILE="/tmp/container_launcher/experiment_data"
+
+getImageDetails() {
+    # image will look like `projects/confidential-space-images/global/images/confidential-space-hardened-230600`
+    #                   or `projects/confidential-space-images/global/images/confidential-space-debug-230600`
```

Comment:
Using the image name won't work for images created by our CI. It's not a big issue since we can copy an image over, but it is more work. Is there a reason we didn't go with something like build time?

Reply:
Yes I did notice this in my testing.
We should talk about this offline because there's no perfect solution here.

Reply:
Will change how our presubmit images are generated as we talked about offline.